_posts/en/newsletters/2022-01-05-newsletter.md

```diff
@@ -0,0 +1,209 @@
+---
+title: 'Bitcoin Optech Newsletter #181'
+permalink: /en/newsletters/2022/01/05/
+name: 2022-01-05-newsletter
+slug: 2022-01-05-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes an alternative proposal to slowly phase
+in full replace-by-fee and announces a series of meetings to review the
+proposed `OP_CHECKTEMPLATEVERIFY` soft fork.  Also included are our
+regular sections with announcements of releases and release candidates
+and summaries of notable changes to popular Bitcoin infrastructure
+projects.
+
+## News
+
+- **Brief full RBF, then opt-in RBF:** Jeremy Rubin [replied][rubin rbf]
+  to an older Bitcoin-Dev mailing list thread described in [Newsletter
+  #154][news154 rbf] about enabling full [replace by fee][topic rbf]
+  (RBF) in Bitcoin Core.  Currently any transaction that signals
+  according to [BIP125][] can be replaced by a higher-feerate
+  alternative (with some exceptions).  The previous proposal was to
+  eventually allow any transaction to be replaced ("full RBF")---not just
+  those that signal that replacability is wanted.  Some merchants have
+  indicated that they prefer that relay nodes make replacability as hard
+  as reasonably possible, at least optionally, so that they can
+  immediately accept unconfirmed transactions in exchange for low-cost
+  goods and services.
+
+    Rubin's alternative still encourages moving to full RBF, but
+    suggests starting by allowing full RBF of any transaction for *n*
+    seconds after it is first received by a node.  After *n* seconds,
+    the BIP125 opt-in flag is honored the same as now.  This can allow
+    merchants to accept unconfirmed transactions like they do now after
+    those *n* seconds have elapsed.  More importantly, it may allow
+    protocols that depend on replacability for security to not have to
```

Comment:
```suggestion
    protocols that depend on replaceability for security to not have to
```

Reply:
@adamjonas thanks for catching these!  I assumed my spell checker just didn't have replaceability in its dict and so hit "Ignore All".  Apparently I had previously trained it and was ignoring it wrongly.